README.md

```diff
@@ -1 +1,39 @@
-# Yahtzee
\ No newline at end of file
+
+<h1>Yahtzee</h1>
```

Comment:
There's nothing wrong with using html headers, but # also works to make headers. For example:

\### Header

is typeset as

### Header